### PR TITLE
Add Travis badge

### LINKED ISSUE #73: Add Travis CI badge to README

### PR DESCRIPTION
**Pull request by aarushgupta**
Fixes #73

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/marco-c/autowebcompat.svg?branch=master)](https://travis-ci.org/marco-c/autowebcompat)
+
 # AutoWebCompat - Automatically detect web compatibility issues
```
Comment (marco-c) [inline README.md]: Can you put it -after- the title?

Comment (aarushgupta) [reply README.md]: yeah sure.

Comment (aarushgupta) [reply README.md]: Done. Sorry about that.

**Comment by aarushgupta** [comment]
Could you please tell me how I can contribute further?

**Comment by marco-c** [comment]
> Could you please tell me how I can contribute further?

You can choose any of the other issues to work on.

**Comment by aarushgupta** [comment]
Will do. Thanks!

**Pull request merged**